File: 프로그래머스/unrated/181836. 그림 확대/그림 확대.cpp
#include <string>
#include <vector>

using namespace std;

vector<string> solution(vector<string> picture, int k) {
    vector<string> answer;
    
    for (int i = 0; i < picture.size(); i++)
    {
        string temp = "";
        for (int j = 0; j < picture[0].size(); j++)
        {
            int l = k;
            while(l--)
                temp.push_back(picture[i][j]);
        }
        int z = k;
        while(z--)
            answer.push_back(temp);
    }
    return answer;
}

File: 프로그래머스/0/181836. 그림 확대/그림 확대.java
class Solution {

    public String[] solution(String[] picture, int k) {
        String[] str = new String[picture.length * k];

        int idx = 0;
        for (String s : picture) {
            StringBuilder sb = new StringBuilder();
            for (String tmp : s.split("")) {
                sb.append(tmp.repeat(k));
            }
            for (int i = 0; i < k; i++) {
                str[idx++] = sb.toString();
            }
        }

        return str;
    }
}